src/components/Profile/MyPosts/MyPosts.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import s from './MyPosts.module.css';
```

Comment:
Consider renaming "s" to "styles" to be more descriptive.

Reply:
/fxd